Makefile

```diff
@@ -54,7 +54,7 @@ export GOPATH GOBIN GO111MODULE DOCKER_CLI_EXPERIMENTAL
 
 # Generate all combination of all OS, ARCH, and OSVERSIONS for iteration
 ALL_OS = linux windows
-ALL_ARCH.linux = amd64
+ALL_ARCH.linux = amd64 arm64
```

Comment:
Adding `arm64` as a separate tag instead of using `--platforms "linux/amd64, linux/arm64"` with buildx to be consistent with current build tags for the driver `driver:<tag>-<os>-<arch>`